client/src/App.js

```diff
@@ -18,39 +18,45 @@ import PatientImmediateAttList from './components/PatientImmediateAttList.compon
 function App() {
   return (
     <Router>
-      <div>
-        <div className="container">
+      
+      <div className="container">
           <br/>
-          <Route path="/">
+          <Route exact path="/">
               <Landing />
-          </Route>
-        </div>
+          </Route> 
+      </div>
       <div className="container">
-          <Route path="/patient/login">
+          <Route exact path="/patient/login">
               <Login />
           </Route>
        </div>
-          <Route path="/provider/login">
+      <div className="container">
+          <Route exact path="/provider/login/:id">
```

Comment:
I do not think /:id needs to be here

Reply:
we need this parameter to display details

Reply:
I believe that is the function of the backend routes